### ISSUE TITLE
add a variable to kickstart with the RHHI configuration

### ISSUE THREAD

**Issue by dthurston** [RedHatGov/lunchbox#1]
I would like to create a variable in lunchbox_disk.yml that will allow you to use the RHHI kickstart file.

**Comment by deversmann** [comment]
Lunchbox_disk.yml is basically just a demo of the role. There is a variable
in the role that lets you choose the KS file, the iso and the location of
the new iso along with some other options. Check out the role docs.

**Comment by dthurston** [comment]
Thanks!  I was just going to add it as a comment so people know it's one of two options for kickstart due to the partitioning. :+1:

**Comment by deversmann** [comment]
Yeah.... I tried to make the role more generic in general.  It will even do Fedora and CentOS.  As for the lunchbox_disk.yml, I don't think it will work for anybody but me.  It's set up to run locally and I have my paths in there.  Should I change that?

**Comment by dthurston** [comment]
it worked for me... I just changed the vars to match my paths.  That should be the first step for anyone who wants to (re)deploy the lunchbox.  I'll contact you offline.
